activestorage/app/models/active_storage/variation.rb

```diff
@@ -66,7 +66,7 @@ def key
     # Applies image transformations using the ImageProcessing gem.
     def image_processing_transform(file, format)
       operations = transformations.inject([]) do |list, (name, argument)|
-        list.tap do |list|
```

Comment:
I prefer to rename one of them. All documentation of `tap` that you look always have a block argument, so I prefer to keep it using the block argument. Or even remove the `tap` entirely, I believe that would make the code clear.

Reply:
Renamed inner to `local_list`.

Unfortunately, removing the `tap` raises:

```Error:
ActiveStorage::VariantTest#test_disabled_variation_of_JPEG_blob:
NoMethodError: undefined method `inject' for nil:NilClass


bin/rails test /home/user/code/rails/activestorage/test/models/variant_test.rb:41
```

Reply:
If you replace `inject` with `each_with_object`, you should be able to remove the call to `tap`:

```ruby
operations = transformations.each_with_object([]) do |(name, argument), list|
```

Reply:
Yes!
Done.
Thank you :angel: